portality/tasks/datalog_journal_added_update.py

```diff
@@ -85,32 +75,15 @@ def to_datalog_journal_added(journal: Journal) -> DatalogJournalAdded:
     return record
 
 
-class LastDatalogJournalAddedQuery:
-    def __init__(self):
-        pass
-
-    def query(self):
-        return {
-            "size": 1,
-            "sort": [
-                {
-                    "date_added": {
-                        "order": "desc"
-                    }
-                }
-            ],
-            "query": {
-                "match_all": {}
-            }
-        }
-
-
-def get_latest_date_added():
-    record = next(DatalogJournalAdded.iterate(LastDatalogJournalAddedQuery().query()), None)
+def get_fetch_datalog_date(n_days=30):
+    record = find_last_datalog()
     if record is None:
-        return datetime.datetime.now()
+        return datetime.datetime(2014, 3, 19)
```

Comment:
I am not clear why this date should be Mar 19 2024. Should it be 1st Feb 2024?
Also I think it would be better if this date is added in setting file instead of hard coding it.

Reply:
that is 2014-03-19, most early date of journal that showed in excel. I don't think that value is configurable and always changed in different environment.

Reply:
Correct, the list should only show journals from February 2024, as it says at the top of the sheet: https://docs.google.com/spreadsheets/d/1Kv3MbgFSgtSDnEGkA2JacrSjunRu0umHeZCtcMeqO5E/edit?usp=sharing - this isn't a change in requirements. It's the way it was from the beginning but at some point it came undone and started pulling in everything.